config/initializers/devise.rb

```diff
@@ -136,6 +136,22 @@
   config.sign_out_via = :get
 
   config.case_insensitive_keys = [:email]
+  config.omniauth :openid_connect, {
+    name: :openid_connect,
+    issuer: "https://login.lescommuns.org",
```

Comment:
It might be worth extracting this provider URL to an ENV var as well?

Reply:
> how this code block will behave if an instance doesn't configure all these OpenID ENV vars

Yeah that's a good catch, I'll test it and add a conditional if necessary.

> It might be worth extracting this provider URL to an ENV var as well?

Possibly... but I think it might be YAGNI, for the foreseeable future we'll want all instances to be using the same provider.